drivers/flash/flash_stm32.c

```diff
@@ -205,7 +224,9 @@ static const struct flash_driver_api flash_stm32_api = {
 #ifdef CONFIG_FLASH_PAGE_LAYOUT
 	.page_layout = flash_stm32_page_layout,
 #endif
-#if defined(CONFIG_SOC_SERIES_STM32F4X)
+#if defined(CONFIG_SOC_SERIES_STM32F0X)
+	.write_block_size = 2,
```

Comment:
Can't we use FLASH_WRITE_BLOCK_SIZE from generated_board.dts.h here? (just questioning here, not blocking comment)

Reply:
Yes, it should use it already, but added it because other platforms had it...

Reply:
ok, let's say this should be done in another PR.
Created #5305 for this purpose